float32 floatVal
---
float32 retVal
